data_steward/tools/consolidated_achilles_report/curation_report/server.py

```diff
@@ -1,15 +1,16 @@
 import SimpleHTTPServer
 import SocketServer
 
-
 PORT = 8000
 
+
 class Handler(SimpleHTTPServer.SimpleHTTPRequestHandler):
     pass
 
+
 Handler.extensions_map['.json'] = 'application/json'
 
 httpd = SocketServer.TCPServer(("", PORT), Handler)
 
-print "serving at port", PORT
+print("serving at port {port}".format(port=PORT))
```

Comment:
Some static analysis (like pylint) will complain about the unnecessary parenthesis.  But I think if we're serious about moving to 3.X, we should leave them.  This will make that a little easier.

Reply:
Going forward, this usage makes our life easier while upgrading our platform. Thats the only reason I am following this convention.

Reply:
Agreed.  Going forward, it makes life easier.  I didn't mean to imply you should change this.